sgl-kernel/csrc/cpu/common.h

```diff
@@ -38,11 +38,40 @@ namespace {
         using packed_t = int8_t;                                                \
         return __VA_ARGS__();                                                   \
       }                                                                         \
+      case at::ScalarType::Float8_e4m3fn : {                                    \
+        using packed_t = at::Float8_e4m3fn;                                     \
+        return __VA_ARGS__();                                                   \
+      }                                                                         \      
       default:                                                                  \
         TORCH_CHECK(false, "Unsupported floating data type.\n");                \
     }                                                                           \
   }()
 
+#define CPU_DISPATCH_PACKED_FLOAT_TYPES(TYPE1, TYPE2, ...)                      \
```

Comment:
won't need this one.

Reply:
Removed in https://github.com/mingfeima/sglang/pull/47/commits/48293178f5f0bb51ca032430f2d5c4a219f3984e